docs/source/docs/installation/installation_prod.inc

```diff
@@ -8,26 +8,21 @@ containerized environment please refer to :ref:`docker_installation`.
 1. Install the MegaQC package
 -----------------------------
 
-MegaQC is available on both the Python Package Index (PyPI) and conda
-(bioconda channel). To install using PyPI, run the following command:
+MegaQC is available on both the Python Package Index (PyPI).
+We are planning to add MegaQC to Conda soon.
+To install using PyPI, run the following command:
 
 .. code:: bash
 
    pip install megaqc[prod]
 
-To install with conda:
-
-.. code:: bash
-
-   conda install -c bioconda megaqc
-
 2. Export environment variables
 -------------------------------
 
 By default, MegaQC runs in development mode with a sqlite flat file
 database (this is to make it as simple as possible to get up and running
 for a quick test / demo). To tell MegaQC to use a production server, you
-need to set the ``MEGAQC_PRODUCTION`` environment variable to true.
+need to set the ``MEGAQC_PRODUCTION`` environment variable to true (=1).
```

Comment:
I would still advise giving an example here. It's not intuitive how to represent a boolean via env vars